deployment/volumes/volume-snapshots/snapshot.yaml

```diff
@@ -2,9 +2,9 @@
   apiVersion: snapshot.storage.k8s.io/v1alpha1
   kind: VolumeSnapshot
   metadata:
-    name: neo4j-data-snapshot
+    name: uploads-snapshot
     namespace: human-connection
   spec:
     source:
-      name: neo4j-data-claim
```

Comment:
What happened to the neo4j snapshot? Anyways, as far as I remember this was supposed to be edited by the developer.